dom0/sd-dom0-files.sls

```diff
@@ -64,20 +64,16 @@ dom0-install-securedrop-workstation-template:
       - file: dom0-workstation-rpm-repo
       - pkg: dom0-remove-securedrop-workstation-stretch-template
 
-# Copy script to system location so admins can run ad-hoc
+# Remove the legacy auto updater script
 dom0-update-securedrop-script:
-  file.managed:
+  file.absent:
     - name: /usr/bin/securedrop-update
-    - source: salt://securedrop-update
-    - user: root
-    - group: root
-    - mode: 755
 
-# Symlink update script into cron, for single point of update
+
+# Remove symlink to legacy updater
 dom0-update-securedrop-script-cron:
-  file.symlink:
+  file.absent:
```

Comment:
Note: may want to enforce removal of `/srv/salt/securedrop-update` as well

Reply:
Nit: since we're now enforcing state=absent for three different files, we can collapse into a single task:

```
dom0-remove-cron-updater-logic:
  file.absent:
    - names:
      - /usr/bin/securedrop-update
      - /etc/cron.daily/securedrop-update-cron
```

Regarding the `/srv/salt/securedrop-update` removal, that's best handled in the "clean" action above. (In prod, such removals will be handled by the RPM package spec.)